pkg/privilege/privileges/ldap/ldap_common.go

```diff
@@ -123,6 +123,8 @@ func (impl *ldapAuthImpl) tryConnectLDAPThroughStartTLS(address string) (*ldap.C
 	err = ldapConnection.StartTLS(&tls.Config{
 		RootCAs:    impl.caPool,
 		ServerName: impl.ldapServerHost,
+		MinVersion: tls.VersionTLS12,
```

Comment:
Should we make it as a config such as grpc?

https://github.com/dfawley/grpc-go/blob/aa13a323eeff39203a2e20ab534d675e2360e425/credentials/tls.go#L176

GRPC makes it as a config. if you don't set it, it will use tls1.2

Reply:
Good point! We do have a config related to TLS version(https://docs-archive.pingcap.com/tidb/v7.2/tidb-configuration-file#tls-version), maybe we should control it using this.

Reply:
let us update this in the next PR.